### PR TITLE
Document stack create command

### LINKED ISSUE #306: Add documentation for appsody stack create
Add doc for https://github.com/appsody/appsody/issues/368

### PR DESCRIPTION
**Pull request by helenmasters**
## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Description
Add documentation for the `appsody stack create` command which is being added to the Appsody CLI.

## Related Issues
  Fixes #306

### CODE REVIEW

File: content/docs/stacks/create.md [hunk 1]
```diff
@@ -4,19 +4,35 @@ title: Creating a Stack
...
-The quickest way to get started creating a new stack is to use the [sample stack](https://github.com/appsody/stacks/tree/master/samples/sample-stack).
+### Using the Appsody CLI
+
+The quickest way to create a new stack is to use the `appsody stack create` command, which creates a new stack, by copying another stack. By default, the new stack is based on the [sample stack](https://github.com/appsody/stacks/tree/master/samples/sample-stack). For example, to create a new stack, named `my-stack`, in a new directory use this command:
+
+`appsody stack creates my-stack`
+
+If you want to use a different stack as the basis for your new stack, use the copy flag to specify the stack you want to use as the starting point. You can use `appsody list` to see the available stacks. For example, to create a new stack, called `my-stack`, based on the Node.js Express stack use this command:
+
+`appsody stack create my-stack –copy incubator/nodejs-express`
```
Comment (neeraj-laad) [inline content/docs/stacks/create.md]: i think this should be `--copy` not `-copy`

Comment (helenmasters) [reply content/docs/stacks/create.md]: Thanks @neeraj-laad, I've fixed this typo and pushed a new commit.

File: content/docs/stacks/create.md [hunk 2]
```diff
@@ -4,19 +4,35 @@ title: Creating a Stack
 
 # Creating a Stack
 
-A stack is made up of two main components, the stack image and templates that
-use the stack's image.
+You might like to create your own stack, perhaps the current Appsody stacks don't meet your needs, maybe you'd like to add a framework or language that isn't currently supported. This doc walks you through how to create a new stack.
+
```
Comment (doveye) [inline content/docs/stacks/create.md]: "Stack" in the heading should be all lowercase.

Suggest changing the first sentence as follows:
> You might like to create your own stack, for example if the current Appsody stacks don't meet your needs or if you want to add a framework or language that isn't currently supported. 

Think this sentence is unnecessary as the heading already tells people this:
> This document walks you through how to create a new stack.

File: content/docs/stacks/create.md [hunk 3]
```diff
@@ -4,19 +4,35 @@ title: Creating a Stack
...
+A stack comprises two main components, a stack image, and templates that use the stack image to provide starter applications for your development team to build on.
+
+You can start by using either, the Appsody CLI `appsody stack create` command, or Git clone. Both involve basing your new stack on another stack, which then becomes the starting point for developing your own Appsody stack.
+
```
Comment (doveye) [inline content/docs/stacks/create.md]: Suggest the following:

> You can start by using either the Appsody CLI `appsody stack create` command, or Git clone. Both methods involve basing your new stack on an existing stack, to create the starting point for developing your own Appsody stack.

File: content/docs/stacks/create.md [hunk 4]
```diff
@@ -4,19 +4,35 @@ title: Creating a Stack
 
 # Creating a Stack
 
-A stack is made up of two main components, the stack image and templates that
-use the stack's image.
+You might like to create your own stack, perhaps the current Appsody stacks don't meet your needs, maybe you'd like to add a framework or language that isn't currently supported. This doc walks you through how to create a new stack.
+
+A stack comprises two main components, a stack image, and templates that use the stack image to provide starter applications for your development team to build on.
+
```
Comment (doveye) [inline content/docs/stacks/create.md]: Suggest a colon:

> A stack comprises two main components: a stack image, and templates that use the stack image to provide starter applications for your development team to build on.

File: content/docs/stacks/create.md [hunk 5]
```diff
@@ -4,19 +4,35 @@ title: Creating a Stack
...
 ## Getting started
 
-The quickest way to get started creating a new stack is to use the [sample stack](https://github.com/appsody/stacks/tree/master/samples/sample-stack).
+### Using the Appsody CLI
+
+The quickest way to create a new stack is to use the `appsody stack create` command, which creates a new stack, by copying another stack. By default, the new stack is based on the [sample stack](https://github.com/appsody/stacks/tree/master/samples/sample-stack). For example, to create a new stack, named `my-stack`, in a new directory use this command:
+
+`appsody stack creates my-stack`
+
```
Comment (doveye) [inline content/docs/stacks/create.md]: Should be "create" not "creates"

File: content/docs/stacks/create.md [hunk 6]
```diff
@@ -4,19 +4,35 @@ title: Creating a Stack
...
+You can start by using either, the Appsody CLI `appsody stack create` command, or Git clone. Both involve basing your new stack on another stack, which then becomes the starting point for developing your own Appsody stack.
+
 ## Getting started
 
-The quickest way to get started creating a new stack is to use the [sample stack](https://github.com/appsody/stacks/tree/master/samples/sample-stack).
+### Using the Appsody CLI
+
+The quickest way to create a new stack is to use the `appsody stack create` command, which creates a new stack, by copying another stack. By default, the new stack is based on the [sample stack](https://github.com/appsody/stacks/tree/master/samples/sample-stack). For example, to create a new stack, named `my-stack`, in a new directory use this command:
+
```
Comment (doveye) [inline content/docs/stacks/create.md]: Suggest the following changes:

> [...] which creates a new stack by copying an existing stack.

> For example, to create a new stack named `my-stack`, in a new directory, use this command:

File: content/docs/stacks/create.md [hunk 7]
```diff
@@ -4,19 +4,35 @@ title: Creating a Stack
...
+`appsody stack creates my-stack`
+
+If you want to use a different stack as the basis for your new stack, use the copy flag to specify the stack you want to use as the starting point. You can use `appsody list` to see the available stacks. For example, to create a new stack, called `my-stack`, based on the Node.js Express stack use this command:
+
```
Comment (doveye) [inline content/docs/stacks/create.md]: Minor comment - could use backticks around "copy" in the text, to distinguish it as a command.

File: content/docs/stacks/create.md [hunk 8]
```diff
@@ -4,19 +4,35 @@ title: Creating a Stack
...
+To create a new stack using Git, clone the [sample stack](https://github.com/appsody/stacks/tree/master/samples/sample-stack) in the stacks repository.
 
 1. Clone the stacks repository:
    ```bash
    git clone https://github.com/appsody/stacks.git
    cd stacks/samples/sample-stack
    ```
 
-You now have the basic [structure](/content/docs/stacks/stack-structure.md) of a stack ready to create an Appsody stack.
+Whichever method you used to create your stack you now have the basic [structure](/content/docs/stacks/stack-structure.md) of an Appsody stack.
```
Comment (doveye) [inline content/docs/stacks/create.md]: Add a comma after "Whichever method you used to create your stack"

File: content/docs/stacks/create.md (hunk 5 above)
Comment (helenmasters) [reply content/docs/stacks/create.md]: Very good spot @doveye! I've addressed all of your review comments (thank you for those), please can you re-review?

Comment (doveye) [reply content/docs/stacks/create.md]: Review approved :)

File: content/docs/stacks/create.md (hunk 6 above)
Comment (neeraj-laad) [reply content/docs/stacks/create.md]: @helenmasters Do we want to include `an existing stack` suggestion from @doveye

Comment (helenmasters) [reply content/docs/stacks/create.md]: @neeraj-laad I've just made the change and commited it.

**Pull request merged**